src/node.ts

```diff
@@ -89,15 +89,13 @@ function compileHook(code: string, filename: string) {
 
 function hookExtensions(exts: readonly string[]) {
   if (piratesRevert) piratesRevert();
-  piratesRevert = addHook(compileHook, { exts: exts as string[], ignoreNodeModules: true });
+  piratesRevert = addHook(compileHook, { exts: exts as string[], ignoreNodeModules: false });
```

Comment:
Here I set `ignoreNodeModules` to `false` in order to match the behavior of Babel. The ignoring `node_modules` behavior will be done by #13 instead.